src/middlewares.js

```diff
@@ -424,6 +432,59 @@ export function promiseEnforceMasterKeyAccess(request) {
   return Promise.resolve();
 }
 
+export const addRateLimit = (route, id) => {
+  const config = Config.get(id);
+  const express = config.expressApp;
+  addLimitForRoute(express, route);
+};
+
+export const handleRateLimit = (api, rateLimit) => {
+  const options = Array.isArray(rateLimit) ? rateLimit : [rateLimit];
+  for (const route of options) {
+    addLimitForRoute(api, route);
+  }
+};
+
+const addLimitForRoute = (api, route) => {
+  api.use(
+    route.path || '*',
```

Comment:
'*' doesn't look like regex, what is this syntax?

Reply:
What does this syntax come from? Is that syntax defined by `express`?

Reply:
I am really not sure. As far as I have used express I have used "*" for match all routes.

https://stackoverflow.com/questions/16637414/how-do-i-redirect-all-unmatched-urls-with-express

Reply:
No default for `requestPath` anyway, so this will be removed

Reply:
From the [docs](https://expressjs.com/en/guide/routing.html):

> Route paths, in combination with a request method, define the endpoints at which requests can be made. Route paths can be strings, string patterns, or regular expressions.
>
> The characters ?, +, *, and () are subsets of their regular expression counterparts. The hyphen (-) and the dot (.) are interpreted literally by string-based paths.

And specifically:

> Express uses [path-to-regexp](https://www.npmjs.com/package/path-to-regexp) for matching the route paths; see the path-to-regexp documentation for all the possibilities in defining route paths. [Express Route Tester](http://forbeslindesay.github.io/express-route-tester/) is a handy tool for testing basic Express routes, although it does not support pattern matching.